app/models/children_content_page.rb

```diff
@@ -0,0 +1,14 @@
+class ChildrenContentPage < Fae::StaticPage
+
+    @slug = 'children_content'
+  
+    # required to set the has_one associations, Fae::StaticPage will build these associations dynamically
```

Comment:
Which associations is this comment referring to?

Reply:
I actually copied the others models, and changed the slug. All of them have this comment there